specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -103,6 +103,17 @@
               "name": "SearchOptions"
             }
           },
+          {
+            "$ref": "#/definitions/Debug",
+            "name": "debug",
+            "in": "query",
+            "type": "string",
+            "description": "This parameter controls debug response information for queries and allows the user to pass in a combination of flags like so: 'Semantic | All'. The values 'semantic' or 'all' are only valid when the query type is 'semantic'. Defaults to 'Disabled'.",
```

Comment:
'All' was intended to be a superset that includes 'Semantic', and would support new categories as this feature evolves. On second thought, I'm thinking about only documenting 'Semantic' and 'Disabled' for now, since they are the only relevant options for a user.